pgsql/bin/functions/do_rewind

```diff
@@ -30,6 +30,9 @@ if [[ "$MASTER_SLAVE_SWITCH" == "1" ]]; then
 
     echo ">>>>>> Restoring configs"
     gosu postgres repmgr standby restore-config -D $PGDATA --config-archive-dir=/tmp/repmgr-archive
+else
```

Comment:
nope. Sounds reasonable enough. just might be easier to move out from the if scope lines 7-8, 28-32. So you don't need else section.